test/controllers/post_controller_test.exs

```diff
@@ -6,6 +6,21 @@ defmodule Tilex.PostControllerTest do
     assert html_response(conn, 200) =~ "Today I Learned"
   end
 
+  test "supports a pagination parameter", %{conn: conn} do
+    conn = get(conn, post_path(conn, :index, page: 1))
```

Comment:
Is `1` a string when it comes in?

Reply:
Yes, it's a string.

Reply:
I think we should change it to a string in the test then. Otherwise you could achieve the same results with an `is_integer` guard, and the tests would pass, but in reality it would fail.